<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration</class>
 <widget class="QDialog" name="Registration">
  <property name="geometry">
   <rect>
    <x>10</x>
    <y>10</y>
    <width>520</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registration</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    background-color: #333333;
    color: #ffffff;
    border: 1px solid #555555;
    border-radius: 5px;
   </string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/regIcon.webp</normaloff>../icons/regIcon.webp</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>160</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>120</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Login (Username):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>120</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>120</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineLogin">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>350</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePass1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>350</width>
     <height>26</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="linePass2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>350</width>
     <height>26</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Registration</receiver>
   <slot>accept()</slot>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Registration</receiver>
   <slot>reject()</slot>
  </connection>
 </connections>
</ui>


